config/puma.rb

```diff
@@ -36,3 +36,19 @@
 
 # Allow puma to be restarted by `rails restart` command.
 plugin :tmp_restart
+
```

Comment:
are you using this?

Reply:
because it was a config suggestion.
create a profile file and send this command